prem_utils/models.json

```diff
@@ -83,6 +83,16 @@
                     "group": "others",
                     "finetuning": true
                 },
+                {
+                    "slug": "command-r-plus",
+                    "model_type": "text2text",
+                    "context_window": 128000,
+                    "input_cost_per_token": 0.000003,
+                    "output_cost_per_token": 0.000015,
+                    "alias": "command-r-plus",
+                    "group": "others",
```

Comment:
should we change the group to 'command'? Now that we have 4 command models, it would be more appropriate, I guess

Reply:
done ✅